lib/i18n/backend/active_record.rb

```diff
@@ -97,9 +97,9 @@ def lookup(locale, key, scope = [], options = {})
             result.first.value
           else
             result = result.inject({}) do |hash, translation|
-              hash.deep_merge build_translation_hash_by_key(key, translation)
+              Utils.deep_merge(hash, build_translation_hash_by_key(key, translation))
```

Comment:
This change will make this gem incompatible with all `i18n` versions prior to `1.9.1` because `I18n::Utils` was introduced only here https://github.com/ruby-i18n/i18n/commit/3104705b8e6dab462fe537dbb0d88c1c01635bf5

Theoretically, we could find a workaround but I don't see any points switching away from `activesupport` as it is one of the `activerecord` dependencies so the main purpose of the https://github.com/ruby-i18n/i18n/commit/3104705b8e6dab462fe537dbb0d88c1c01635bf5 is no longer valid

Reply:
You are right! After all, i18n is not always used with active-record, but this gem does!